src/app/CommandSender.h

```diff
@@ -286,6 +286,8 @@ class CommandSender final : public Messaging::ExchangeDelegate
                   bool aSuppressResponse = false) :
         CommandSender(static_cast<Callback *>(nullptr), apExchangeMgr, aIsTimedRequest, aSuppressResponse)
     {}
+    CommandSender(bool aIsTimedRequest = false, bool aSuppressResponse = false);
```

Comment:
This does not seem like a usable CommandSender... how would it report results?

Reply:
you may just remove bool aIsTimedRequest = false, bool aSuppressResponse = false

Reply:
Removed the new constructor I added and used Optional.